spec/models/payment_spec.rb

```diff
@@ -44,4 +44,46 @@
       end
     end
   end
+
+  describe "validations of claims’ personal details" do
```

Comment:
Can we make this better express the business logic, rather than the code's behaviour? Say something like:

```suggestion
  describe "when the payment is for more than one claim" do
```

Reply:
Changed it to a `context` plus your suggested text.